Sequencer.cs

```diff
@@ -16,39 +16,99 @@ namespace WordSequence
 {
     public class Sequencer : CustomPwGenerator
     {
-        private string GetConfigurationPath()
+        /* if returnEmptyConfig is true, or a user config file is found, returns
+         * the path to the user config file.
+         *
+         * if returnEmptyConfig is false, or the user config file path is not
+         * specified, and the user config file is not found, returns the path to
+         * the global config.
+         *
+         * Note reading the global config at startup (for a missing user config
+         * file) and then writing the user config after changing settings, allows
+         * a default configuration to be specified in the global config.
+         */
+        private string GetConfigurationPath(bool returnEmptyConfig)
         {
-            string config = System.Configuration.ConfigurationManager.AppSettings["configPath"];
+            string config = System.Configuration.ConfigurationManager.AppSettings["userConfigPath"];
 
-            return System.IO.Path.GetFullPath(config);
+            if (null != config)
+            {
+                config = Path.Combine(
+                        Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
+                        config);

```

Comment:
Things gets confusing here as the userConfigPath may already contain a path; maybe we can split the configuration into two, ie a path and a filename?  Otherwise, detect if the path actually contains a path or only a filename...